third-party/realdds/include/realdds/dds-stream.h

```diff
@@ -0,0 +1,81 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <functional>
+#include <memory>
+#include <string>
+
+namespace realdds {
+
+
+// Represents a stream of information (images, motion data, etc..) from a single source received via the DDS system.
+// A stream can have several profiles, i.e different data frequency, image resolution, etc..
+class dds_stream
+{
+public:
+    struct profile
+    {
+        int16_t uid;       // Stream unique ID
+        int16_t framerate; // FPS
+        int8_t type;       // Kind of video, motion etc..
```

Comment:
For the following 3 enums like ints IMO we  need to add a matching enum in a comment or use an internal enum
It will be hard to understand the valid values otherwise..

Reply:
I think that they should remain as general as possible, we don't want to have to sync values with LibRS.

Reply:
It's about to change anyway right? so we will look at it again later